string old_path # e.g. /objects/container_grape1
string new_path # e.g. /entities/robot/containers/container_tower1
---
bool success # e.g. result: /entities/robot/containers/container_tower1/container_grape_1/...